cmd/argo-actions/main.go

```diff
@@ -37,20 +39,21 @@ func main() {
 	logger, err := logger.New(cfg.Logger)
 	exitOnError(err, "while creating zap logger")
 
-	client := local.NewDefaultClient(cfg.LocalHubEndpoint)
+	localClient := local.NewDefaultClient(cfg.LocalHubEndpoint)
+	publicClient := public.NewDefaultClient(cfg.PublicHubEndpoint)
```

Comment:
Actually, the download client does not have validation of TypeInstnace so passing the Public client is not needed but I improved others to have hubClient instead of Local/Public client.